languages/swift/concepts/generic-functions/links.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Maybe include the https://docs.swift.org/swift-book/LanguageGuide/Generics.html#ID181 link here?